text/0060-tasks-api.md

```diff
@@ -150,6 +152,14 @@ List of global tasks by `type`:
 | canceledTasks | The number of tasks successfully canceled. If the task fails, `0` is displayed. `null` when the task status is enqueud or processing. |
 | originalQuery | The extracted URL query parameters used in the originating task cancelation request. |
 
+##### taskDeletion
+
+| Name          | Description |
+|---------------|-------------|
+| matchedTasks  | The number of tasks that can be deleted based on the request. If the API key doesn’t have access to any of the indexes specified in the request via the `indexUid` query parameter, those tasks will not be included in `matchedTasks`. `null` when the task status is enqueued. |
```

Comment:
`matchedTasks` is not `null` when the task status is `enqueued`. The matching is done at enqueue-time

Reply:
Thanks @loiclec